src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.html

```diff
@@ -63,7 +63,11 @@ <h1 mat-dialog-title>GitHub App Logs</h1>
           </div>
           <div *ngSwitchCase="'success'">
             <th mat-header-cell *matHeaderCellDef mat-sort-header>Success</th>
```

Comment:
Wondering if we should rename the Success column to Status because there are 4 possible values now

Reply:
Renamed "Success" column to "Status".